Model/Service/DumpCmsDataService.php

```diff
@@ -54,6 +55,7 @@ public function execute(array $types, ?array $identifiers)
         $varDirectory = $this->filesystem->getDirectoryWrite(DirectoryList::VAR_DIR);
         $varPath = $this->directoryList->getPath(DirectoryList::VAR_DIR);
         $workingDirPath = $varPath . '/sync_cms_data';
+        $this->blocksMapping = $this->getBlocksMapping();
```

Comment:
Don't do DB calls inside of __construct() method, __construct should only be used to initialize the variables, not populate them.